### DISCUSSION TITLE
Journal Papers or White Papers Utilizing Magma

### DISCUSSION THREAD

**Discussion by tayloreisman16** [magma/magma#7138] [General]
Hey guys, I'm trying to find some research pertaining to what people have done with Magma. I've only been able to find one relevant paper that is more of a survey over the different open-source 5G implementations. Do you guys have any relevant papers off the top of your head?

**Comment by ekowtaylor** [comment]
@PARitter for initial comments

**Comment by karthiksubraveti** [comment]
@tayloreisman16 there was a lot of talks given during our developer conference. Check it out
https://www.magmacore.org/blog/magma-dev-conf-videos/

**Reply by tayloreisman16** [reply]
Alright, thank you.
